google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/prepare/base_tag_factory.py

```diff
@@ -18,12 +18,11 @@
 from google.protobuf import timestamp_pb2
 import six
 
+from google.datacatalog_connectors.commons import prepare
+
 
 class BaseTagFactory:
-    __UTF8_CHARACTER_ENCODING = 'UTF-8'
-    # String field values are limited to 2000 bytes size when encoded in UTF-8.
```

Comment:
This comment refers to the `__STRING_VALUE_UTF8_MAX_LENGTH`, so it shouldn't be deleted.

Reply:
The constant name and the comment are basically the same. Do you really think that comment helps?

Reply:
Makes sense, it can be deleted. 👍🏼